networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/AbstractBlobSidecarsValidator.java

```diff
@@ -20,19 +20,19 @@
 import tech.pegasys.teku.spec.Spec;
 import tech.pegasys.teku.spec.datastructures.blobs.versions.deneb.BlobSidecar;
 
-public class AbstractBlobSidecarsValidatingProxy {
+public class AbstractBlobSidecarsValidator {
 
   private static final Logger LOG = LogManager.getLogger();
 
   protected final Spec spec;
   protected final KZG kzg;
 
-  public AbstractBlobSidecarsValidatingProxy(final Spec spec, final KZG kzg) {
+  public AbstractBlobSidecarsValidator(final Spec spec, final KZG kzg) {
     this.spec = spec;
     this.kzg = kzg;
   }
 
-  protected boolean verifyBlobSidecar(final BlobSidecar blobSidecar) {
+  boolean verifyBlobSidecarKzg(final BlobSidecar blobSidecar) {
```

Comment:
can rename to `validateBlobSidecarKzg` for consistency

Reply:
I was trying to make the most different name to validator to not mess up what to call from validator

Reply:
sure, makes sense